src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -494,6 +493,10 @@
     <Error Condition="!Exists('..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets'))" />
   </Target>
   <Import Project="..\..\packages\Microsoft.Azure.DocumentDB.1.9.2\build\Microsoft.Azure.DocumentDB.targets" Condition="Exists('..\..\packages\Microsoft.Azure.DocumentDB.1.9.2\build\Microsoft.Azure.DocumentDB.targets')" />
+  <Target Name="CopyAzureFunctionsToNodeModules" AfterTargets="Build">
```

Comment:
So this copies the node_modules folder to the output directory, and it will be part of our normal site extension packaging?

Note we also support running the ScriptHost w/o our site extension. We produce a WebJobs.Script.Host.zip file as part of our build artifacts that contains all the requisite files to run. These files would have to be part of that as well.